azext_iot/deviceupdate/providers/base.py

```diff
@@ -90,7 +93,7 @@ def get_data_client(self, endpoint: str, instance_id: str) -> DeviceUpdateClient
 
         profile = Profile()
         client: DeviceUpdateClient = DeviceUpdateClient(
-            credential=profile.get_login_credentials(resource=AUTH_RESOURCE_ID)[0],
+            credential=profile.get_login_credentials()[0],
```

Comment:
Should this use `profile.get_raw_token(...)` ?

Reply:
Possibly, having some trouble in initial testing using it so I'm not sure, it worked in my testing so I'm inclined to leave it for now, might be worth another look in our next round of updates.